nsv13/code/game/objects/items/custom_guns.dm

```diff
@@ -139,3 +139,21 @@
 	ammo_type = /obj/item/ammo_casing/shotgun/buckshot
 	max_ammo = 6
 	multiple_sprites = 1
+
+/obj/item/gun/ballistic/rifle/boltaction/pdc
+	name = "\improper Point Defence Rifle"
+	desc = "A rifle made to fire PDC rounds."
+	icon = 'nsv13/icons/obj/guns/projectile.dmi'
+	icon_state = "pdcrifle"
+	item_state = "pdcrifle"
```

Comment:
If you're not going to add a custom back sprite I recommend using one of a different gun, so it's at least recognizable on the character when equipped. (I'd say that just using the mosin one is fine, as it is similar in functionality and damage)
To just use the default /rifle/boltaction back sprite you can remove the item_state:
```suggestion
```